snowpack/src/commands/dev.ts

```diff
@@ -268,6 +268,14 @@ export async function startServer(commandOptions: CommandOptions) {
   const filesBeingDeleted = new Set<string>();
   const filesBeingBuilt = new Map<string, Promise<SnowpackBuildMap>>();
 
+  logger.debug(`Using in-memory cache for mount points:`, {
```

Comment:
nit: what do you mean by in-memory cache? We do use an in-memory cache, but it's unrelated to the mounting logic. 

If you think that's relevant info, maybe instead:

```
logger.debug(`using in-memory cache`);
logger.debug(`mounting directories:`, {task ...
```

Reply:
Yes, I think that wording would work. Coming from other web-dev-servers, I expected to find a directory somewhere that had all the files that the local server was serving. I turned debug info on expecting to find such a place, but then realized, oh, duh, this is snowpack and I bet it's doing something super fast and not on disk ;)